domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -25,6 +25,7 @@ enum class ReviewScheduleCalculationRule {
   PRISONER_READMISSION,
   PRISONER_TRANSFER,
   BETWEEN_RELEASE_AND_3_MONTHS_TO_SERVE,
+  BETWEEN_3_MONTHS_AND_3_MONTHS_7_DAYS_TO_SERVE,
   BETWEEN_3_AND_6_MONTHS_TO_SERVE,
```

Comment:
Arguably this enum value `BETWEEN_3_AND_6_MONTHS_TO_SERVE` is now incorrect.
The pendant in me is saying it should really be called `BETWEEN_3_MONTHS_8_DAYS_AND_6_MONTHS_TO_SERVE` .... but I really don't know if I can be bothered / know whether it's worth it.

If I were to rename it I would need to rename it here in the domain, in the corresponding entity enum, and in the swagger spec for the resultant REST API enum.

The value is rendered out in one of the REST API responses. Our UI will be consuming that API, but I dont think the verboseness of this value will be an issue to us as this level of detail belongs in our application anyway (and chances are our UI will probably map it to some text that makes it clear anyway)
But (via the integration api), MN will be consuming an API that might contains this value. (we haven't designed it yet). And I don't know if renaming it will make it too verbose / implementation specific for MN (or even if they care about it? 🤔 )

What do you think?

Reply:
If we were going to rename it, now would be the time to do it ....

Reply:
It might be pedantic but it describes the situation and shows that we've considered the situation. So I think the enum should be BETWEEN_3_MONTHS_8_DAYS_AND_6_MONTHS_TO_SERVE even though it is a whopper! 

I don't think anyone will care though so it's not urgent.

Reply:
I know it's not urgent, but I think if the pedant in both of us is calling us to do it, then I'm going to do the rename now 😁